python/amici/setuptools.py

```diff
@@ -0,0 +1,134 @@
+"""Helper functions for AMICI core and module package preparation"""
+
+import os
+import platform
+import shlex
+
+try:
+    import pkgconfig # optional
+except ModuleNotFoundError:
+    pkgconfig = None
+
+
+def getBlasConfig():
+    """
+    Find CBLAS-compatible BLAS
+
+    Arguments:
+
+    Returns:
+
+    Raises:
+
+    """
+
+    blaspkgcfg = {'include_dirs': [],
+                  'library_dirs': [],
+                  'libraries': [],
+                  'define_macros': [],
+                  'extra_compile_args' : [],
+                  'extra_link_args' : []
+                  }
+
+    if platform.system() == 'Linux':
+        blaspkgcfg['libraries'] = ['cblas']
+
+    if 'BLAS_CFLAGS' in os.environ:
+        blaspkgcfg['extra_compile_args'].extend(shlex.split(os.environ['BLAS_CFLAGS']))
+
+    if 'BLAS_LIBS' in os.environ:
+        blaspkgcfg['extra_link_args'].extend(shlex.split(os.environ['BLAS_LIBS']))
+
+    return blaspkgcfg
+
+
+def getHdf5Config():
+    """Find HDF5 include dir and libs
+
+    Arguments:
+
+    Returns:
+
+    Raises:
+
+    """
+    if pkgconfig:
+        h5pkgcfg = pkgconfig.parse('hdf5')
+        # NOTE: Cannot use pkgconfig.exists('hdf5f'), since this is true although
+        # no libraries or include dirs are available
+        h5pkgcfg['found'] = 'include_dirs' in h5pkgcfg and h5pkgcfg['include_dirs']
+        if h5pkgcfg['found']:
+            return h5pkgcfg
+
+    h5pkgcfg = {'include_dirs': [],
+                'library_dirs': [],
+                'libraries': [],
+                'define_macros': []
+                }
+
+    # try for hdf5 in standard locations
+    hdf5_include_dir_hints = ['/usr/include/hdf5/serial',
+                              '/usr/local/include',
+                              '/usr/include', # travis ubuntu xenial
+                              '/usr/local/Cellar/hdf5/1.10.2_1/include'] # travis macOS
+    hdf5_library_dir_hints = ['/usr/lib/x86_64-linux-gnu/', # travis ubuntu xenial
+                              '/usr/lib/x86_64-linux-gnu/hdf5/serial',
+                              '/usr/local/lib',
+                              '/usr/local/Cellar/hdf5/1.10.2_1/lib'] # travis macOS
+
```

Comment:
standard for windows?

Reply:
Not sure there is anything remotely standard on Windows. If you have something, we can add it..

Reply:
Well, the Program Files (x86) folders, but we can see that whence hdf5 for windows works.